frontend/src/views/Market.vue

```diff
@@ -1746,6 +1746,14 @@ export default Vue.extend({
 
     maxPrecisionSkill(listedPrice: string): string {
       return this.convertStringToDecimal(this.convertWeiToSkill(listedPrice), 8);
+    },
+
+    upperFirstChar(input: string): string {
```

Comment:
this is more of a utility function and should be elsewhere, not here, but it's not super important